packages/wrangler/src/pages/build.tsx

```diff
@@ -136,27 +134,25 @@ export async function buildFunctions({
 	onEnd,
 	plugin = false,
 	buildOutputDirectory,
-	directory,
+	routesOutputPath,
 	nodeCompat,
-}: {
```

Comment:
The arguments were implicitly depending on `PagesBuildArgs`. This change makes it explicitly depend on fields from `PagesBuildArgs`